bundles/org.openhab.binding.revogismartstripcontrol/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.4-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.revogismartstripcontrol</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: RevogiSmartStripControl Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <scope>provided</scope>
+      <version>2.9.10</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.9.10</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <scope>provided</scope>
+      <version>2.9.10</version>
+    </dependency>
```

Comment:
These dependencies should already be available to you through the jersey-all library that is included by the parent.

Reply:
> Please address as many of these warnings as you can:

I added some extra null checks, but the warnings stay the same. Obviously this check doesn't know short-circuit-evaluation and doesn't see that we sometimes just return if the the object is null

Reply:
Without these dependencies I was not able to build and run my add-on.

At the moment I cannot test it, because it doesn't compile locally. I get this message on every mvn command I do for my addon:

```
Failed to execute goal org.commonjava.maven.plugins:directory-maven-plugin:0.3.1:directory-of (directories) on project org.openhab.binding.revogismartstripcontrol: Cannot find directory for project: org.openhab.addons:org.openhab.addons.reactor -> [Help 1]
```

Reply:
And to be honest, at the moment this process of integrating a new add-on feels very annoying. It's not motivating at all.

Reply:
Leave the these dependencies in for now, we can try and address them later. A failure to build is definitely not what I expected.

Reply:
Leave the these dependencies in for now, we can try and address them later. A failure to build is definitely not what I expected.